examples/porting/lv_port_lcd_stm32_template.h

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file lv_port_lcd_stm32_template.h
+ *
+ */
+
+/*Copy this file as "lv_port_disp.h" and set this value to "1" to enable content*/
+#if 0
+
+#ifndef LV_PORT_LCD_STM32_TEMPL_H
+#define LV_PORT_LCD_STM32_TEMPL_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#if defined(LV_LVGL_H_INCLUDE_SIMPLE)
+#include "lvgl.h"
+#else
+#include "lvgl/lvgl.h"
+#endif
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/**********************
+ * GLOBAL PROTOTYPES
+ **********************/
+/* Initialize low level display driver */
+void lv_port_disp_init(void);
```

Comment:
As we can have other STM port files too (e.g. LTDC) let name it
```suggestion
void lv_port_lcd_stm32_init(void);
```

Reply:
This has been copied from lv_port_disp_template.c. Since 99% of the users will only use a single display, it does not really matter if the templates use the same function name (actually, it makes sense). But since this is just a template, the users can change it if there is any conflict.